<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LexicalAnalysisProcedureClass</class>
 <widget class="QMainWindow" name="LexicalAnalysisProcedureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LexicalAnalysisProcedure</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="file_name_alabel">
        <property name="text">
         <string>正则表达式</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="file_name_alabel_2">
        <property name="text">
         <string>源程序代码</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="run_pushButton">
        <property name="text">
         <string>分析</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="get_source_file_pushButton">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="source_file_edit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="regular_file_edit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="get_regular_file_pushButton">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="clear_pushButton">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="nfa_tab">
         <attribute name="title">
          <string>NFA</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="nfa_area">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>681</width>
               <height>344</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="dfa_tab">
         <attribute name="title">
          <string>DFA</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="dfa_area">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>681</width>
               <height>344</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mindfa_tab">
         <attribute name="title">
          <string>最小化DFA</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="mindfa_area">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>681</width>
               <height>344</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="total_tab">
         <attribute name="title">
          <string>DFA汇总</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="program_tab">
         <attribute name="title">
          <string>词法分析程序</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="program_edit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="code_tab">
         <attribute name="title">
          <string>单词编码</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="encode_edit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
